retrieve-logs/README.md

```diff
@@ -0,0 +1,77 @@
+# How to replay missing Matomo requests to Matomo
+
+These steps describe how to get missing Matomo requests from AWS CloudWatch.
+Once obtained, the logs can be used to replay the requests to Matomo.
+
+You will need to authenticate with AWS via the command line to use this script.
+Namely, you will need to set the following environment variables to appropriate values:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_SESSION_TOKEN` - this is obtained from the AWS Security Token Service
+- `AWS_DEFAULT_REGION`
+
```

Comment:
Might be worth including a link to the `assume-role.py` script in the `verify-event-infrastructure` repo to do this or give example of AWS CLI command.

Also, any specific IAM role required?

Reply:
I'm still trying to figure out exactly what permissions are needed. I think it's just `logs:StartQuery` and `logs:GetQueryResults`. I think I'll postpone addressing this until we've confirmed these if that's ok.